bundle/container-jfr-operator.v0.3.1.clusterserviceversion.yaml

```diff
@@ -205,11 +205,13 @@ spec:
                 serviceAccountName: container-jfr-operator
                 containers:
                   - name: container-jfr-operator
-                    image: 'quay.io/rh-jmc-team/container-jfr-operator:0.3.0'
+                    image: 'quay.io/andrewazores/container-jfr-operator:0.3.1'
```

Comment:
This should go back to rh-jmc-team when you're done with local testing.

Reply:
Yep.